xml/decoders/Piko_SmartDecoder_4.1_Sound_Next_18_40501-18.xml

```diff
@@ -1,1958 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@uli-2016 
Why was this file deleted? Was it intentional?

Reply:
Yes, its correct, because this decoder is now included in the new decoder file 'Piko_SmartDecoder_4.1_Sound.xml'